sup3r/bias/bias_transforms.py

```diff
@@ -0,0 +1,195 @@
+# -*- coding: utf-8 -*-
+"""Bias correction transformation functions."""
+import numpy as np
+import logging
+from warnings import warn
+from rex import Resource
+
+
+logger = logging.getLogger(__name__)
+
+
+def global_linear_bc(input, scalar, adder, out_range=None):
+    """Bias correct data using a simple global *scalar +adder method.
+
+    Parameters
+    ----------
+    input : np.ndarray
+        Sup3r input data to be bias corrected, assumed to be 3D with shape
+        (spatial, spatial, temporal) for a single feature.
+    scalar : float
+        Scalar (multiplicative) value to apply to input data.
+    adder : float
+        Adder value to apply to input data.
+    out_range : None | tuple
+        Option to set floor/ceiling values on the output data.
+
+    Returns
+    -------
+    out : np.ndarray
+        out = input * scalar + adder
+    """
+    out = input * scalar + adder
+    if out_range is not None:
+        out = np.maximum(out, np.min(out_range))
+        out = np.minimum(out, np.max(out_range))
+    return out
+
+
+def local_linear_bc(input, feature_name, bias_fp, lr_padded_slice,
+                    out_range=None):
+    """Bias correct data using a simple global *scalar +adder method.
```

Comment:
Edit for local method

Reply:
like site-by-site, right? yes, good catch